app/test/features/comments/add_comment_test.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:acter/common/providers/common_providers.dart';
+import 'package:acter/features/comments/widgets/add_comment_widget.dart';
+import 'package:acter_avatar/acter_avatar.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_easyloading/flutter_easyloading.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import '../../common/mock_data/mock_avatar_info.dart';
+import 'mock_data/mock_comments_manager.dart';
+
+void main() {
+  late MockCommentsManager mockCommentsManager;
+  late MockAvatarInfo mockAvatarInfo;
+
+  setUp(() {
+    mockCommentsManager = MockCommentsManager();
+    mockAvatarInfo = MockAvatarInfo();
+  });
+
+  testWidgets('should display avatar and comment input',
+      (WidgetTester tester) async {
+    await tester.pumpWidget(
```

Comment:
use `tester.pumpProviderWidget` to avoid all the unnecessary widget setup.

(here and in the others)